flow/connectors/utils/monitoring/monitoring.go

```diff
@@ -317,3 +317,32 @@ func UpdateRowsSyncedForPartition(ctx context.Context, pool *pgxpool.Pool, rowsS
 	}
 	return nil
 }
+
+func DeleteMirrorStats(ctx context.Context, pool *pgxpool.Pool, flowJobName string) error {
+	_, err := pool.Exec(ctx, `DELETE FROM peerdb_stats.qrep_partitions WHERE flow_name ILIKE 'clone\_' || replace(replace($1, '_', '\_'), '%', '\%') || '\_%'`, flowJobName)
```

Comment:
Runs into the case where two mirrors are substrings of each other
Need a new column - `mirror_name` or `parent_mirror` - in the qrep tables

Reply:
Fixed and updated description